doc/user_guide/getting_started.rst

```diff
@@ -37,7 +37,7 @@ Use the following command to create a new project:
 .. code-block:: shell
 
    cookiecutter https://github.com/exasol/python-toolbox.git \
-     --directory project-template
+    --checkout <latest-tag> --directory project-template
```

Comment:
The default behavior without a specifier is to use `main`. Changes on main may not yet be released, which could lead to breaking changes due in the usage of the PTB. The project-template uses a released version of the PTB for its CI and dependencies.